src/Middleware/Localization/src/RequestLocalizationOptions.cs

```diff
@@ -84,6 +84,12 @@ public RequestCulture DefaultRequestCulture
         /// </example>
         public bool FallBackToParentUICultures { get; set; } = true;
 
+        /// <summary>
+        /// Gets or sets the value of the reponse Content-Language header to a <see cref="CultureInfo.CurrentUICulture"/>.
+        /// Defaults to <c>false</c>
+        /// </summary>
+        public bool SetContentLanguageHeader { get; set; } = false;
```

Comment:
I think `SetContentLanguageResponseHeader` is more descriptive

> No need to assign it to false

Yes no need to assign `false`

Reply:
What about `EnableContentLanguageResponseHeader`?

Reply:
> Doesn't enable the header.

You're right it adds instead, `ApplyCurrentCultureToContentLanguageResponseHeader` is too long, I think `AddContentLanguageResponseHeader` is a appropriate because the intend is adding the header by toggling the property on or off